src/socketio.quiver.js

```diff
@@ -315,103 +316,102 @@ QuiverSocialProvider.prototype.finishLogin_ = null;
  *
  * @override
  */
-QuiverSocialProvider.prototype.login = function(loginOpts, continuation) {
+QuiverSocialProvider.prototype.login = function(loginOpts) {
   if (this.isOnline_()) {
-    continuation(undefined, this.err("LOGIN_ALREADYONLINE"));
-    return;
+    return Promise.reject('LOGIN_ALREADYONLINE');
```

Comment:
I think in freedom you're supposed to reject with an object of a particular form, hence the `this.err` helper function.